Bottom: 5b49675e8097b435f15f907123f01b761ced736d
Top:    6b7ed9be4aa4132fbdc9aa0762198786d84af015
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-29 09:31:34 -0500

Refresh of enable-secrets

---
diff --git a/roles/buildbot-worker-ntel/tasks/main.yml b/roles/buildbot-worker-ntel/tasks/main.yml
index 91e7d6d4..d96081aa 100644
--- a/roles/buildbot-worker-ntel/tasks/main.yml
+++ b/roles/buildbot-worker-ntel/tasks/main.yml
@@ -67,6 +67,8 @@
   copy:
     content: "{{ known_hosts }}\n"
     dest: '{{ secrets_hostdir }}/{{ secrets_known_hosts }}'
+    group: '{{ buildbot_uid }}'
+    owner: '{{ buildbot_uid }}'
 
 - name: Checkout image repo
   git:
diff --git a/roles/buildbot-worker-ptxdist/tasks/main.yml b/roles/buildbot-worker-ptxdist/tasks/main.yml
index e9499970..e86c19a4 100644
--- a/roles/buildbot-worker-ptxdist/tasks/main.yml
+++ b/roles/buildbot-worker-ptxdist/tasks/main.yml
@@ -67,6 +67,8 @@
   copy:
     content: "{{ known_hosts }}\n"
     dest: '{{ secrets_hostdir }}/{{ secrets_known_hosts }}'
+    group: '{{ buildbot_uid }}'
+    owner: '{{ buildbot_uid }}'
 
 - name: Checkout image repo
   git:
